packages/calypso-build/package.json

```diff
@@ -72,13 +72,13 @@
 		"terser-webpack-plugin": "^4.2.2",
 		"thread-loader": "^2.1.3",
 		"typescript": "^4.0.3",
-		"webpack": "^4.44.2",
-		"webpack-cli": "^3.3.12"
+		"webpack-cli": "^4.2.0"
 	},
 	"peerDependencies": {
 		"enzyme": "^3.11.0",
 		"jest": ">=26.4.0",
 		"react": "^16.0.0",
-		"react-dom": "^16.0.0"
+		"react-dom": "^16.0.0",
+		"webpack": "^5.10.3"
```

Comment:
`webpack` as peer could cause trouble for projects that depend on `calypso-build` but don't install `webpack` on their own. Then the package manager, at least npm 6, will issue a warning about a missing peer dep, but won't install it.

npm 7 will do the install and will just check that the parent package doesn't depend on a conflicting version of `webpack`. I don't know about Yarn.

On the other hand, for consumers that have their own `webpack` (and use only the provided configs), peer dep would be a perfect fit.